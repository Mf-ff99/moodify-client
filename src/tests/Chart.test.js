import React from 'react'
import Chart from '../components/Chart'

import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`Chart component`, () => {
    it('Renders without crashing', () => {
        shallow(<Chart />)
    })
    it('renders form approriately', () => {
        const wrapper = shallow(<Chart />)
        expect(toJson(wrapper)).toMatchSnapshot()
      })
})