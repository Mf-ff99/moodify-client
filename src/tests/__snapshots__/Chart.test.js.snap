// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart component renders form approriately 1`] = `
<div
  id="chart-js"
>
  <select
    name="filter"
    onChange={[Function]}
  >
    <option
      value="pastWeek"
    >
      Past week
    </option>
    <option
      value="pastMonth"
    >
      Past month
    </option>
    <option
      value="allTime"
    >
      All time
    </option>
  </select>
  <br />
  <Line
    data={
      Object {
        "datasets": Array [
          Object {
            "backgroundColor": "rgba(75,192,192,1)",
            "borderColor": "rgba(0,0,0,1)",
            "borderWidth": 2,
            "data": Array [
              0,
            ],
            "fill": true,
            "label": "your mood",
            "lineTension": 0.7,
          },
        ],
        "labels": Array [
          "Monday",
          "Tuesday",
          "Wednesday",
          "Thursday",
          "Friday",
          "Saturday",
          "Sunday",
        ],
        "moods": Array [],
      }
    }
    options={
      Object {
        "legend": Object {
          "display": true,
          "fontSize": 20,
          "position": "top",
        },
        "maintainAspectRatio": false,
        "responsive": true,
        "title": Object {
          "display": true,
          "fontSize": 20,
          "text": "Your mood history",
        },
      }
    }
  />
</div>
`;
